packages/commons-ui-core/src/ImageButton/ImageButton.js

```diff
@@ -57,11 +57,13 @@ ImageButton.propTypes = {
    * The image URL.
    */
   src: PropTypes.node,
+  onClick: PropTypes.func,
```

Comment:
```suggestion
```

No need to specify this... if one wants to use `onClick`, they can use it. If they want to use `href`, they can use that too. Anything that `ButtonBase` support should be supported.